Cargo.toml

```diff
@@ -89,46 +89,3 @@ panic = "unwind"
 inherits = "release"
 lto = "fat"
 codegen-units = 1
-
-# Reason: We need to patch substrate dependency of frontier to our fork
```

Comment:
This only works temporarily because you have changed dependencies in frontier to point to our fork, we'll likely bring it back in the next upgrade